src/routes/contacts/+page.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+  import LabelInput from '$lib/component/LabelInput.svelte';
+  import Graph from './Graph.svelte';
+  import Icon from 'svelte-icons-pack/Icon.svelte';
+  import BiMap from 'svelte-icons-pack/bi/BiMap';
+</script>
+
+<main class="-10 my-20 flex flex-col justify-center ">
```

Comment:
```suggestion
<main class="-10 my-20 flex flex-col justify-center">
```
Linter doesn't catch this? 🤔